src/CoinMpsIO.hpp

```diff
@@ -17,6 +17,11 @@
 #include "CoinPackedMatrix.hpp"
 #include "CoinMessageHandler.hpp"
 #include "CoinFileIO.hpp"
+
+#ifdef COINUTILS_HAS_GLPK
+#include "glpk.h"
+#endif
```

Comment:
Same as in Osi and Clp: This means that any code that includes CoinMpsIO.hpp needs to be able to find glpk.h when compiling. This shouldn't be necessary.

Reply:
As I commented in Osi, I see that we may not need to include `glpk.h` here, but the includes are all in files that themselves call `readGmpl`, which means that in order to compile `CoinMpsIO.cpp`, we have to be pulling in a header that itself pulls in `glpk.h`. In other words, if `glpk.h` can't be found, then the build is going to break with or without this particular include.

Reply:
We do actually need this because readGMPL is implemented in CoinMpsIO (I had forgotten that).

Reply:
The comment here is on `CoinMpsIO.hpp`, a header that is installed. So it isn't just about the build of CoinMpsIO.cpp, but also the build of any project that includes `CoinMpsIO.hpp`.